bedrock/settings/__init__.py

```diff
@@ -264,7 +265,6 @@
 
 # support older browsers (mainly Safari)
 CSP_FRAME_SRC = CSP_CHILD_SRC
-CSP_FONT_SRC = _csp_font_src
```

Comment:
This nonetheless contains the defaults explicitly, so after adding the defaults without smarter merging the output gets a bit repetitive:

`Content-Security-Policy: default-src 'self' *.mozilla.net *.mozilla.org *.mozilla.com *.allizom.org; […]; font-src 'self' *.mozilla.net *.mozilla.org *.mozilla.com *.allizom.org 'self'; […]`

which is not a real issue, as any duplicate value is used only on its first occurrence but it can make some validators angry (possible warning level). The rule is a tad uglier for pocket mode. I'll clean it up a bit…